docs/install.rst

```diff
@@ -1,15 +1,70 @@
 Installation
 ============
 
-This installation guide includes only the GeoCAT-examples installation instructions.
-Please refer to `GeoCAT Contributor's Guide <https://geocat.ucar.edu/pages/contributing.html>`_
-for installation of the whole GeoCAT project.
+GeoCAT-examples is a collection of standalone visualization examples, and it is not distributed
+as a conda package; thus, there is no installation for it (via conda or other package management systems).
+The easiest way to access GeoCAT-examples scripts as a whole is by cloning the repo and then using a
+`Conda <http://conda.pydata.org/docs/>`_ environment.
 
-Installing GeoCAT-examples
---------------------------
+For information about installing the entire GeoCAT project, see our
```

Comment:
Just as in GeoCAT-comp and others, I'd start this paragraph with something like "This installation guide includes only the GeoCAT-examples ... instructions."

Reply:
I slightly reworded this section. Please let me know what you think

Reply:
Sounds good!